Libplanet/Net/Swarm.cs

```diff
@@ -168,6 +167,32 @@ static Swarm()
                 _privateKey.PublicKey.ToAddress(),
                 _appProtocolVersion,
                 _logger);
+
+            _requests = new AsyncCollection<MessageRequest>();
+            _runtimeCancellationTokenSource = new CancellationTokenSource();
+            _runtimeProcessor = Task.Run(() =>
+            {
+                // Ignore NetMQ related exceptions during NetMQRuntime.Dispose() to stabilize tests
+                try
+                {
+                    using (var runtime = new NetMQRuntime())
+                    {
+                        runtime.Run(
+                            ProcessRuntime(_runtimeCancellationTokenSource.Token),
+                            ProcessRuntime(_runtimeCancellationTokenSource.Token),
+                            ProcessRuntime(_runtimeCancellationTokenSource.Token),
+                            ProcessRuntime(_runtimeCancellationTokenSource.Token),
+                            ProcessRuntime(_runtimeCancellationTokenSource.Token)
```

Comment:
What are these lines for?

Reply:
These lines initialize runtimes to process request in `Swarm<T>`.  (it needs to allow concurrency for request processing.)